components/Table.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { styled, CSS, StitchesVariants } from '../stitches.config';
+
+import type * as Polymorphic from '@radix-ui/react-polymorphic';
+
+const DEFAULT_TAG = 'table';
+
+export const Caption = styled('caption', {
+  textAlign: 'start',
+  marginBottom: '$5',
+});
+
+export const Tbody = styled('tbody', {
+  width: '100%',
+});
+
+export const Tfoot = styled('tfoot', {
+
+});
+
+export const Tr = styled('tr', {
+
+});
+
+export const Th = styled('th', {
+  fontWeight: 'unset',
+  textAlign: 'start',
+  fontSize: '$2',
+  py: '$2',
+  borderBottom: '1px solid $gray300',
+  variants: {
+    align: {
+      start: {
+        textAlign: 'start',
+      },
+      center: {
+        textAlign: 'center',
+      },
+      end: {
+        textAlign: 'end',
+      }
+    },
+    border: {
+      solid:{
+        borderBottom: '1px solid $gray300',
+      },
+      dashed: {
+        borderBottom: '1px dashed $gray700',
+      }
+    },
+  },
+  defaultVariants: {
+    align: 'start',
+    border: 'solid',
+  }
+});
+
+export const Td = styled('td', {
+  py: '$2',
+  borderBottom: '1px solid $gray300',
+  fontSize: '$2',
+  variants: {
+    align: {
+      start: {
+        textAlign: 'start',
+      },
+      center: {
+        textAlign: 'center',
+      },
+      end: {
+        textAlign: 'end',
+      }
+    },
+    border: {
+      solid:{
+        borderBottom: '1px solid $gray300',
+      },
+      dashed: {
+        borderBottom: '1px dashed $gray700',
+      }
+    },
+  },
```

Comment:
I think this border should go on the Tr element
Otherwise browsers mess up the rhythm on the dashed border

<img width="443" alt="Screenshot 2021-05-13 at 19 27 39" src="https://user-images.githubusercontent.com/8441036/118155926-6e287e00-b421-11eb-9689-fec882baa133.png">